docs/content/en/docs/tasks/cluster/cluster-upgrades/baremetal-upgrades.md

```diff
@@ -0,0 +1,225 @@
+* Cert-manager
+* Etcdadm CAPI provider
+* EKS Anywhere controllers and CRDs
+* GitOps controllers (Flux) - this is an optional component, will be upgraded only if specified
+
+The latest versions of these core EKS Anywhere components are embedded into a bundles manifest that the CLI uses to fetch the latest versions 
+and image builds needed for each component upgrade. 
+The command detects both component version changes and new builds of the same versioned component.
+If there is a new Kubernetes version that is going to get rolled out, the core components get upgraded before the Kubernetes
+version. 
+Irrespective of a Kubernetes version change, the upgrade command will always upgrade the internal EKS
+Anywhere components mentioned above to their latest available versions. All upgrade changes are backwards compatible.
+
+### Check upgrade components
+Before you perform an upgrade, check the current and new versions of components that are ready to upgrade by typing:
+
+```bash
+eksctl anywhere upgrade plan cluster -f cluster.yaml --kubeconfig test-eks-a-cluster.kubeconfig
+```
+
+The output should appear similar to the following:
+
+```
+Worker node group name not specified. Defaulting name to md-0.
+Warning: The recommended number of control plane nodes is 3 or 5
+Worker node group name not specified. Defaulting name to md-0.
+Checking new release availability...
+NAME                     CURRENT VERSION                 NEXT VERSION
+EKS-A                    v0.0.0-dev+build.1000+9886ba8   v0.0.0-dev+build.1105+46598cb
+cluster-api              v1.0.2+e8c48f5                  v1.0.2+1274316
+kubeadm                  v1.0.2+92c6d7e                  v1.0.2+aa1a03a
+vsphere                  v1.0.1+efb002c                  v1.0.1+ef26ac1
+kubadm                   v1.0.2+f002eae                  v1.0.2+f443dcf
+etcdadm-bootstrap        v1.0.2-rc3+54dcc82              v1.0.0-rc3+df07114
+etcdadm-controller       v1.0.2-rc3+a817792              v1.0.0-rc3+a310516
+```
+To the format output in json, add `-o json` to the end of the command line.
+
+### Check hardware availability
+
+Next, you must ensure you have enough available hardware for the rolling upgrade operation to function. This type of upgrade requires you to have one spare hardware server for control plane upgrade and one for each worker node group upgrade. Check [prerequisites]({{< relref "baremetal-upgrades/#prerequisites" >}}) for more information.
+Available hardware could have been fed to the cluster as extra hardware during a prior create command, or could be fed to the cluster during the upgrade process by providing the hardware CSV file to the [upgrade cluster command]({{< relref "baremetal-upgrades/#upgrade-cluster-command" >}}).
+
+To check if you have enough available hardware for rolling upgrade, you can use the `kubectl` command below to check if there are hardware with the selector labels corresponding to the controlplane/worker node group and without the `ownerName` label. 
```

Comment:
```suggestion
To check if you have enough available hardware for rolling upgrade, you can use the `kubectl` command below to check if there are hardware items with the selector labels corresponding to the controlplane/worker node group and without the `ownerName` label. 
```
Not sure if that's necessary, but it seemed to me like "hardware items" or "hardware entries" would make it a bit better.

Reply:
'objects' would be technically correct